### DISCUSSION TITLE
How do you show your media which was recently updated?

### DISCUSSION THREAD

**Discussion by NiklasBr** [jellyfin/jellyfin#11207] [Q&A]
Is there a way to create a playlist which shows the latest media sorted by those which had their metadata updated most recently?

**Comment by JPVenson** [comment]
Hey,

There is no build in way to do that, playlist/collection curation is done mostly manually. 
You can sort for latest added media though:
![image](https://github.com/jellyfin/jellyfin/assets/6794763/c50bf949-365d-4b71-b60f-8f63a8863080)

**Reply by NiklasBr** [reply]
Okay, the reason I need recently _updated_ specifically (as opposed to added) is that it looks like a bunch of items in my library may have been oddly tagged by an automated tool and I need to find them.

**Reply by JPVenson** [reply]
i see, but there is no way for JF to show this because i dont think we even track when a media's metadata was last updated. If you have NFO saver enabled maybe try to sort for last write time?
